src/SFA.DAS.ApprenticeCommitments/Application/Commands/ConfirmEmployerCommand/ConfirmEmployerCommand.cs

```diff
@@ -24,20 +20,4 @@ public class ConfirmEmployerCommand : IUnitOfWorkCommand
         public long ApprenticeshipId { get; }
         public bool EmployerCorrect { get; }
     }
-
```

Comment:
The reason I like having the command definition and the handler in the same file is that it makes it really easy to navigate to the logic code from the controller.  Just F12 on the command passed into Mediatr, and the file that opens up contains the handler code too.

Reply:
I can understand that logic, but we can also just click the sync button and find the next file (I know it's an additional action). 

It still seems (very slight) wrong to name the file XXXXCommand.cs (If we called it XXXCommandAndHandler.cs, but that's long winded or perhaps if we put the command in the Handler file this would be better as the command is really a child class of the handler. 

In saying all of this, the general noise I get from people is that classes should be in their own files. (Not that I agree with that).  But which ever way we do it we should try to be consistent.

Reply:
It doesn't bother me at all that the file is called "Command" and contains the handler, or the other way round - "CommandHandler.cs" containing the command definition would also be fine - I'm happy if you prefer it that way.  My opinion is that files should be small and contain one class, or a couple of highly-related classes.  

I've seen people go to lengths to work around "one-class-per-file" rules - e.g. making the command an embedded type of the handler.  In fact, the inventor of Mediatr does that himself - command, response, verification and handler classes all in one file.  

I'll leave it up to you 😄